cmd/env.go

```diff
@@ -43,6 +43,6 @@ var envCmd = &cobra.Command{
 }
 
 func init() {
-	envCmd.PersistentFlags().BoolVar(&showSensitive, "show-sensitive", false, "Show sensitive/secret environment variables")
+	envCmd.PersistentFlags().BoolVar(&showSensitive, "show-sensitive", false, "Shows sensitive/secret environment variables")
```

Comment:
Admittedly I'm the one to blame for the source text here, but consider changing this to "Displays the value of sensitive/secret environment variables" instead. I feel like the definition of this is unclear until used, in hindsight.